docs/EXTENDED-ARCHIVES.md

```diff
@@ -267,22 +287,10 @@ For example:
 }
 ```
 
-### `.meta/publisher(2x)?.[svg|png]`
-
-A graphic that represents the image publisher.
-
-### `.meta/image(2x)?.[svg|png]`
-
-A graphic that represents the image.
-
 ### `.meta/schema.json`
 
 A [Reconfix][reconfix] image configuration schema.
 
-### `.meta/image.bmap`
-
-A [bmap][bmap] file for the extended archive image.
-
 ### `.meta/instructions.markdown`
 
 A markdown file including post-flash instructions.
```

Comment:
"containing" is probably better than "including" here, as "including" implies it might include other things too ;)